lib/Encoder.ts

```diff
@@ -15,10 +17,12 @@ export default class Encoder {
 
   /**
    * Decodes the given Base64URL string into a Buffer.
-   *
-   * Turns the encoded string to regular base 64 and decode as buffer
    */
-  public static decodeAsBuffer (encodedContent: string): Buffer {
+  public static decodeAsBuffer (encodedContent: string, inputContextForErrorLogging: string): Buffer {
+    if (!Encoder.isBase64UrlString(encodedContent)) {
+      throw new IonError(ErrorCode.EncodedMultiHashIncorrectEncoding, `Given ${inputContextForErrorLogging} must be base64url string.`);
```

Comment:
C&P error: Rename `EncodedMultiHashIncorrectEncoding`, encoded string doesn't have to be a multihash anymore.